Кнопки увелечения и уменьшения - изменил;

viaya-ya opened the pull request:
Переделал, чтобы при нажатии на геопозицию выдавалась окно с данными долготы и широты

viaya-ya committed:
Кнопки увелечения и уменьшения - изменил;
Переделал, чтобы при нажатии на геопозицию выдавалась окно с данными долготы и широты

ptz2018 commented on src/js/App.js:
```diff
@@ -85,7 +85,7 @@ function App() {
                                 }
                             >
-                                <RPopup trigger={"hover"} className="example-overlay">
+                                <RPopup trigger={"onClick"} className="example-overlay">
```
would it work with just click?

ptz2018 commented on src/js/config.js:
```diff
@@ -1,2 +1,2 @@
-const BASE_URL = 'http://localhost:8090';
+const BASE_URL = 'http://localhost:8080';
```
please revert

viaya-ya committed:
Кнопки увелечения и уменьшения - изменил;
Переделал, чтобы при нажатии на геопозицию выдавалась окно с данными долготы и широты

viaya-ya committed:
Кнопки увелечения и уменьшения - изменил;
Переделал, чтобы при нажатии на геопозицию выдавалась окно с данными долготы и широты

viaya-ya committed:
Кнопки увелечения и уменьшения - изменил;
Переделал, чтобы при нажатии на геопозицию выдавалась окно с данными долготы и широты

viaya-ya committed:
Кнопки увелечения и уменьшения - изменил;
Переделал, чтобы при нажатии на геопозицию выдавалась окно с данными долготы и широты
